Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Lite_-_Incident_Handling.yml

```diff
@@ -2174,6 +2174,33 @@ inputs:
   required: false
   description: XDR instance domain.
   playbookInputQuery:
+inputSections:
+- inputs:
+  - Username
+  - incident_id
+  - Hostname
+  - EndpointID
+  - XDRDomain
+  name: Incident Management
+  description: Incident management settings and data, including escalation processes, user engagements, and ticketing methods
+- inputs:
+  - InternalRange
+  name: Enrichment
+  description: Enrichment settings and data, including assets and indicators enrichment using third-side enrichers.
```

Comment:
do you mean third-party (instead of third-side)

Reply:
done